tests/009-test-follow-symlinked-devices.sh

```diff
@@ -0,0 +1,48 @@
+source $SRCDIR/libtest.sh
+
+test_follow_symlinked_devices() {
+  local devs dev
+  local devlinks devlink
+  local test_status=1
+  local testname=`basename "$0"`
+  local vg_name="dss-test-foo"
+
+  # Create a symlink for a device and try to follow it
+  for dev in $TEST_DEVS; do
+    if [ ! -h $dev ]; then
+      devlink="$dev-test.$$"
+      ln -s $dev $devlink
```

Comment:
Another minor nit. Can we create these soft links in some dir like /tmp. Right now they seem to be created in cwd of process and if cleanup fails for some reason, it will just leave persistent links behind.